src/lambdas/fusionauth-site-origin-handler/README.md

```diff
@@ -0,0 +1,17 @@
+# fusionauth-site-origin-handler
+
+This is a Lambda@Edge function that handles redirects for fusionauth.dev.
```

Comment:
```suggestion
This is a Lambda@Edge function that handles redirects for fusionauth.dev and fusionauth.io.
```

Reply:
I only have .dev in there since this is the development branch and it seems to be very out of sync with master.